pvlib/bifacial/loss_models.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
+import pandas as pd
+
+from typing import Literal
+
+
+def power_mismatch_deline(
+    rmad,
+    model: Literal[
+        "fixed-tilt", "single-axis-tracking"
+    ] = "single-axis-tracking",
+    fillfactor: float = None,
+):
+    r"""
+    Estimate DC power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    Depending on the mounting type, the power loss is estimated with either
+    equation (11) or (12) of [1]_. Passing a custom polynomial is also valid.
+
+    Use ``fillfactor`` to account for different fill factors between the
+    trained model and the module of interest.
+    For example, if the fill factor of the module of interest is
+    0.65, then set ``fillfactor=0.65``.
+
+    .. versionadded:: 0.11.0
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the cell-by-cell total
+        irradiance. [Unitless]
+        Check out the *Notes* section for the equation to calculate it from the
+        bifaciality and the front and back irradiances.
+
+    model : str, numpy.polynomial.polynomial.Polynomial or list, default ``"single-axis-tracking"``
+        The model coefficients to use.
+        If a string, it must be one of the following:
+
+        * ``"fixed-tilt"``: Eq. (11) of [1]_.
+        * ``"single-axis-tracking"``: Eq. (12) of [1]_.
+
+        If a :external:class:`numpy.polynomial.polynomial.Polynomial`,
+        it is evaluated as is.
+
+        If neither a string nor a ``Polynomial``, it must be the coefficients
+        of a polynomial in ``rmad``, where the first element is the constant
+        term and the last element is the highest order term. A
+        :external:class:`~numpy.polynomial.polynomial.Polynomial`
+        will be created internally.
+
+    fillfactor : float, optional
+        Fill factor at standard test condition (STC) of the module.
+        Accounts for different fill factors between the trained model and the
+        module under non-uniform irradiance.
+        Models from [1]_ were calculated for a ``fillfactor`` of 0.79.
+        This parameter will only be used if ``model`` is a string.
+        Raises a ``ValueError`` if the model is a custom polynomial.
+        Internally, this argument applies :ref:`Equation (7) <fillfactor_eq>`.
+
+    Returns
+    -------
+    loss : numeric, pd.Series if ``rmad`` is a pd.Series
+        The power loss.
```

Comment:
```suggestion
        The fractional power loss. [unitless]
```

This assumes we change the output to be unitless instead of percent